app/assets/stylesheets/govuk_publishing_components/components/_column-list.scss

```diff
@@ -0,0 +1,20 @@
+.gem-c-column-list {
+  columns: 1;
+
+  @include govuk-media-query($from: tablet) {
+    columns: 2;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    columns: 3;
```

Comment:
I was musing on whether this column number might benefit from being configurable and whether it might be useful for pages such as https://www.gov.uk/government/ministers#ministers-by-department - and also on whether the list items should be able to contain images. It doesn't seem appropriate though as those examples are all just flowing left to right rather than splitting into columns.

Reply:
Of all the possible future extensions this one makes the most sense. The example as you say it's more row based than column based but it's probably still something worth exploring.